docs/getting-started.md

```diff
@@ -1,44 +1,46 @@
-* A TypeScript editor (you can use notepad if you want but I use [alm 🌹](http://alm.tools). Also [lots of other IDES support it as well]( https://github.com/Microsoft/TypeScript/wiki/TypeScript-Editor-Support))
+* TypeScript derleyicisi (Açık Kaynak Kodlu olarak dağıtılmaktadır. Kodlarina [buradan](https://github.com/Microsoft/TypeScript/) erişebilirsiniz. Eğer NPM üzerinden kurmak için [buraya](https://www.npmjs.com/package/typescript) tıklayabilirsiniz.)
+* Bir TypeScript editörü (Notepad ya da diğer bir çok editörden bir tanesini kullanabilirsiniz. Örneğin bulut tabanlı olan [alm](http://alm.tools)'nin güzel bir örnek olması yanında [bir çok farklı editör de mevcuttur]( https://github.com/Microsoft/TypeScript/wiki/TypeScript-Editor-Support).)
 
 
 ![](https://raw.githubusercontent.com/alm-tools/alm-tools.github.io/master/screens/main.png)
 
 
-## TypeScript Version
+## TypeScript Sürümleri
 
-Instead of using the *stable* TypeScript compiler we will be presenting a lot of new stuff in this book that may not be associated with a version number yet. I generally recommend people to use the nightly version because **the compiler test suite only catches more bugs over time**.
+Kitapta anlatacağımız bazı yeni özellikler henüz *kararlı* bir TypeScript derleyicisi tarafından desteklenmemektedir. Hatta bazıları henüz bir sürüm numarası ile bile ilişkilendirilmemiş olabilir.
 
-You can install it on the command line as
+TypeScript derleyicisini aşağıdaki komut ile bilgisayarınıza kurabilirsiniz,
 
 ```
 npm install -g typescript@next
 ```
 
-And now the command line `tsc` will be the latest and greatest. Various IDEs support it too, e.g.
+Bundan sonra artık komut satırında `tsc` komutu son ve en iyi versiyonu olarak kullanılabilir olacaktır. Bir çok editör bu sürümü desteklemektedir örneğin,
 
-* `alm` always ships with the latest TypeScript version.
-* You can ask vscode to use this version by creating `.vscode/settings.json` with the following contents:
+* `alm` her zaman son sürümünde en güncel TypeScript sürümü ile yayınlanmaktadır.
+* Eğer `vscode` kullanıyorsanız aşağıdaki içeriğe sahip bir `.vscode/settings.json` dosyası oluşturursanız son sürümü kullanacaktır:
 
 ```json
 {
   "typescript.tsdk": "./node_modules/typescript/lib"
 }
 ```
 
-## Getting the Source Code
-The source for this book is available in the books github repository https://github.com/basarat/typescript-book/tree/master/code most of the code samples can be copied into alm and you can play with them as is. For code samples that need additional setup (e.g. npm modules), we will link you to the code sample before presenting the code. e.g.
+## Örneklerin Kaynak Kodlarını İndirmek
+Kitabın içerisindeki örneklerin kaynak kodlarına github üzerinden https://github.com/Codefiction/typescript-book/tree/master/code adresinden ulaşabilirsiniz. Kodların büyük bir kısmı alm'ye kopyalandığında hemen çalıştırılabilir durumdadir.
```

Comment:
Repository içerisine bu şekilde link vermek GitHub tarafından önerilmiyor: 
https://github.com/blog/1395-relative-links-in-markup-files

Kitabın içerisindeki örneklerin kaynak kodlarına [github üzerinden](code) ulaşabilirsiniz.